std/http/file_server.ts

```diff
@@ -99,25 +72,7 @@ function fileLenToString(len: number): string {
   return `${(len / base).toFixed(2)}${suffix[suffixIndex]}`;
 }
 
-function createDirEntryDisplay(
-  name: string,
-  url: string,
-  size: number | null,
-  mode: number | null,
-  isDir: boolean
-): string {
-  const sizeStr = size === null ? "" : "" + fileLenToString(size!);
-  return `
-  <tr><td class="mode">${modeToString(
-    isDir,
-    mode
-  )}</td><td>${sizeStr}</td><td><a href="${url}">${name}${
-    isDir ? "/" : ""
-  }</a></td>
-  </tr>
-  `;
-}
-
+// TODO: load files using template
```

Comment:
What does this mean? I would opt that you remove this TODO.

Reply:
if we don’t need to render markdown/code pages, I will remove this todo